packages/app-store/applecalendar/README.mdx

```diff
@@ -3,6 +3,4 @@ items:
   - /api/app-store/applecalendar/1.jpg
 ---
 
-<Slider items={items} />
```

Comment:
This slider was only used in V1. The new version uses the items array in the MDX meta.